GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/PrefetchVerbTests.cs

```diff
@@ -140,9 +140,12 @@ private void PostFetchJobShouldComplete()
             ProcessResult midxResult = GitProcess.InvokeProcess(this.Enlistment.RepoRoot, "multi-pack-index verify --object-dir=\"" + objectDir + "\"");
             midxResult.ExitCode.ShouldEqual(0);
 
-            ProcessResult graphResult = GitProcess.InvokeProcess(this.Enlistment.RepoRoot, "commit-graph read --object-dir=\"" + objectDir + "\"");
-            graphResult.ExitCode.ShouldEqual(0);
-            graphResult.Output.ShouldContain("43475048"); // Header from commit-graph file.
+            if (this.fileSystem.FileExists(Path.Combine(objectDir, "info", "commit-graph")))

```

Comment:
I think we could include a comment about why this is a conditional check... maybe something along the lines of:

```
// A commit graph is not always generated, but if it is, then we want to ensure it is in a good state
```

To indicate to readers why the check is conditional